### PR TITLE
Added filter against *.orig

### PR DESCRIPTION
**Pull request by jcjimenez**
Super simple.

### CODE REVIEW

**Review by c-w — APPROVED** [review]
Is .orig some sort of backup file on Mac?

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,8 @@
 # macOS
 .DS_Store
 
+**/*.orig
```
Comment (Smarker) [inline .gitignore]: What about just using `*.orig`?

**Comment by jcjimenez** [comment]
@c-w it's actually a file git generates when something goes wrong during merging - I assume it's to help one figure out the before/after of a particular file.

File: .gitignore (hunk 1 above)
Comment (jcjimenez) [reply .gitignore]: `*.orig` will capture files at the base of the project directory, so adding `**/` in front of that will ensure that any files no matter their parent directory should be ignored (e.g. src/main/java/com/some/package/SomeClass.scala.orig)

**Pull request merged**